fractal_server/app/routes/aux/_runner.py

```diff
@@ -13,7 +13,7 @@ def _check_backend_is_slurm():
     """
     settings = Inject(get_settings)
     backend = settings.FRACTAL_RUNNER_BACKEND
-    if backend != "slurm":
+    if backend not in ["slurm", "local_processes"]:
```

Comment:
Let's rename this function with `_is_shutdown_available`.

Do we use it anywhere apart from when we stop jobs? If so, let's have two functions: `_is_shutdown_available` and `_is_backend_slurm`.

Reply:
The function is just used in
- `fractal_server/app/routes/api/v[12]/job.py::stop_job`
- `fractal_server/app/routes/admin/v[12].py::stop_job`